turbinia/config/turbinia_config_tmpl.py

```diff
@@ -63,6 +63,9 @@
 # host to properly lock the resource state file.
 RESOURCE_FILE_LOCK = '%s.lock' % RESOURCE_FILE
 
+# For Kubernetes infrastructure. Indicates whether a given pod is set to be deleted.
+TO_BE_DELETED = '%s/turbinia-to-delete.lock' % OUTPUT_DIR
```

Comment:
So this will write a non-unique name file to the OUTPUT_DIR. We need to make 100% sure that this OUTPUT folder is *not* shared with the host, else we get issues with running multiple workers on the same host.

Is OUTPUT_DIR non-shared?

Reply:
Good catch, it's actually shared. Went ahead and updated the `LOCK_FILE` and `TO_BE_DELETED` file to write to `TMP_DIR` instead.